source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/organization-select/display.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Dropdown } from 'semantic-ui-react';
+
+import { OrganizationResource } from '@data';
+import { attributesFor, ORGANIZATIONS_TYPE, idFromRecordIdentity } from '@data';
+import { withTranslations, i18nProps } from '@lib/i18n';
+
+interface IOwnProps {
+  className?: string;
+  organizations: OrganizationResource[];
+  onChange: (value: string) => void;
+}
+
+export type IProps =
+  & IOwnProps
+  & i18nProps;
+
+class Display extends React.Component<IProps> {
+  onChange = (e, dropdownEvent) => {
+    e.preventDefault();
+
```

Comment:
Remove this space to be consistent with render. Perhaps is time to add some linting rules about spacing we both have different styles

Reply:
what space? consistent with render? I'm confused

Reply:
the space between preventDefault and this.props.onChange

Reply:
I think what I have here should be the default. Otherwise it's squished